src/views/homepage/components/learn-section/index.tsx

```diff
@@ -51,17 +52,16 @@ export default function LearnSection({
           className={s.tutorialsList}
           style={
             {
-              '--tutorials-count': tutorials.length,
+              '--tutorials-count': learnCards.length,
             } as CSSProperties
           }
         >
-          {tutorials.map((tutorial: TutorialCardProps) => {
+          {learnCards.map((cardProps: FeaturedLearnCard) => {
+            const { id, type } = cardProps
             return (
-              <li
-                key={slugify(tutorial.heading, { lower: true })}
-                className={s.tutorialsListItem}
-              >
-                <TutorialCard {...tutorial} />
+              <li key={id} className={s.tutorialsListItem}>
+                {type === 'collection' && <CollectionCard {...cardProps} />}
+                {type === 'tutorial' && <TutorialCard {...cardProps} />}
```

Comment:
⛏️ Nitpick: This feels a little odd to me since it feels technically possible that nothing would render here. One approach could be to do like `const CardComponent = type === 'collection' ? CollectionCard : TutorialCard` in the `map` body, and then render `<CardComponent {...cardProps} />`.

For context: I think we had [a Slack thread](https://hashicorp.slack.com/archives/C02T80JTNS2/p1649872956699629), where it felt like we had alignment on wanting to avoid `&&` for conditional rendering (here's [an article I like on the pitfalls of &&](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx))... though I think we still haven't set anything in stone, so definitely feel free to ignore this if it doesn't feel productive 👍